my-react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -29,4 +30,9 @@ class Board extends React.Component {
   }
 }
 
+Board.propTypes = {
+  squares: PropTypes.arrayOf(PropTypes.string),
```

Comment:
This `isRequired` too (the array, not each element).
Add `.isRequired` after `arrayOf`, se we require the array but let each array element be `undefined`/`null` if it has to